torch/_inductor/lowering.py

```diff
@@ -1716,6 +1716,20 @@ def new_empty_strided(
     )
 
 
+@register_lowering(prims.copy_strided.default)
+def copy_strided(x, stride):
+    output = empty_strided(
+        x.get_size(),
+        stride,
+        dtype=x.get_dtype(),
+        layout=torch.strided,
+        device=x.get_device(),
+        pin_memory=False,
+    )
+    copy_(output, x)
```

Comment:
We shouldn't do this with a copy.  In the common case (a FlexibleLayout) we can do this without copying.  It would be something similar to:
https://github.com/pytorch/pytorch/blob/3e6978172e86a97dc4c248406d2b2509a837299f/torch/_inductor/ir.py#L2602-L2634